src/settings.ts

```diff
@@ -103,6 +118,29 @@ export class WikipediaSearchSettingTab extends PluginSettingTab {
 
 		if (settings.additionalTemplatesEnabled) {
 			containerEl.createEl("h2", { text: "Additional Templates" });
+			
+			new Setting(containerEl)
+				.setName("Default path for created notes")
+				.setDesc("Folder where created notes should be saved. Set empty to use the vault root folder. (type to search)")
+					.addDropdown((dropdown: DropdownComponent) => {
+						dropdown
+							.addOptions(this.app.vault.getAllLoadedFiles()
+								.filter(f => (f instanceof TFolder))
+								.reduce((acc, item) => {
+									acc[item.name] = item.name;
+									return acc;
+								}, {} as Record<string, string>))
```

Comment:
This isn't a good solution. Directories with the same name aren't distinguishable...

Reply:
Yeah thats true. I tried to use the path as a display element for the option, but the dropdown element then increases immeasurably the deeper you go into a folder branch. That's actually a shame. I've seen the FolderSuggest from Periodic Notes many times and used it myself in other plugins, but it has additional dependencies and sometimes linting errors. What would be your approach?

Reply:
Otherwise I´ll just add the FolderSuggester...